core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -294,6 +294,10 @@ private void forceResolveRemainingErrors(
                   if (!error.getRegion().isOnField()) {
                     return false;
                   }
+                  if (error.messageType.equals("PASS_NULLABLE") && error.getRegion().isOnField()) {
```

Comment:
Isn't the second condition already guaranteed by the check right above? (i.e. at this point `error.getRegion().isOnField()` is known to be `true` or we would have hit the early return on line 295)

Reply:
Yes, that makes sense, removed it. Thank you c3b2464c2006a93255f580fa94b926d7ae60f31c